models/aave/optimism/aave_optimism_sources.yml

```diff
@@ -30,12 +31,110 @@ sources:
           description: "Liqudity index value of the reserve"
         - name: liqudityRate
           description: "Liqudity rate value of the reserve"
-        - name: reserve
+        - &reserve
+          name: reserve
           description: "Aave reserve contract address"
         - name: stableBorrowRate
           description: "Stable borrow rate value of the reserve"  
         - name: variableBorrowIndex
           description: "Variable borrow index value of the reserve"
         - name: variableBorrowRate
-          description: "Variable borrow rate value of the reserve"   
+          description: "Variable borrow rate value of the reserve"     
+
+      - name: Pool_evt_Supply
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving deposits for aave v3 on optimism"
+        columns:
+        - &amount
+          name: amount
+          description: "The token transafer amount"       
+        - *contract_address 
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - name: onBehalfOf
+          description: "The user wallet address if the token is deposited on behalf of another wallet"
+        - name: referralCode
+          description: "Deposit referral number"  
+        - *reserve
+        - &user
+          name: user
+          description: "The user wallet address"
+
+      - name: Pool_evt_Withdraw
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving withdrawals for aave v3 on optimism"
+        columns:
+        - *amount       
+        - *contract_address 
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - *reserve
+        - name: to
+          description: "Wallet address where the tokens are withdrawn and transferred"
+        - *user

```

Comment:
are `to` and `user` usually the same?

Reply:
Usually yes, but in many cases, users withdraw tokens to another wallet So I feel this column can be useful.